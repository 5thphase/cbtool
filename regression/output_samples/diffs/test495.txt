|execute_script_name |~/cloudbench/scenarios/scripts/openstack_multitenant.sh|/path/to/tool/regression/..//regression/scripts/execute_on_staging.sh
Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command ""/path/to/tool/lib/operations//../../cbact" --procid=TEST_msilva --osp=port:xxxx,experiment_inst:TEST_msilva,host:xxx.xxx.xxx.xxx ,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:xxxx,password:cbpwdZ9,timeout:240,host:xxx.xxx.xxx.xxx ,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,execute_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --daemon". The process id is xxxx.
 status: Counter "VM ARRIVING" equals 1 after 0.088730096817 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 1.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 6.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 7.08873009682 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 8.08873009682 seconds (the counter is assumed to be decreasing).
Waited 8.08873009682 seconds until "VMs ARRIVING" was equal to 0.
|Redis Version |3.0.6 
|VMs APPLICATION ERRORS |5 
