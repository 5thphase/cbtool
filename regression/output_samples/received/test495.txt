running shell command: "rm /tmp/cb*_was_used_on_execution"....
stdout:
 

The global object "vm_defaults" on Cloud TESTCLOUD was modified:
|"sub-attribute" (key)                |old value                          |new value                          
|execute_script_name                  |execute_on_staging.sh              |/home/msilva/cloudbench/regression/..//regression/scripts/execute_on_staging.sh
Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,execute_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=F7804CA2-4B8B-58F5-8DD8-C995DFE6A74F --daemon". The process id is 12269.

Going to wait until the value on counter "VM ARRIVING" is equal to 0 (currently it is equal to 1) waiting 1.0 seconds between samples. The counter is assumed to be decreasing.
 status: Counter "VM ARRIVING" equals 1 after 1 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 6 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 7 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 8 seconds (the counter is assumed to be decreasing).
Waited 8 seconds until "VMs ARRIVING" was equal to 0.
running shell command: "ls /tmp/cb*_was_used_on_execution"....
stdout:
 /tmp/cb_vapp_TESTCLOUD_vm_13_F7804CA2-4B8B-58F5-8DD8-C995DFE6A74F.json_was_used_on_execution


The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |150.172.15.31   |simhosta1                  |SUT                 |none    |none       |0DA072BA-80BB-5C1A-9225-434AE3B53E7C
|vm_3          |tinyvm              |pico32      |155.1.237.121   |simhostb1                  |SUT                 |none    |none       |D6F4A052-6D0F-5AFB-B250-122D99E8D469
|vm_4          |db2                 |copper32    |184.25.167.99   |simhostc1                  |SUT                 |none    |none       |114FBCB9-628F-5D34-AB91-1E08A89C1C73
|vm_5          |netclient           |platinum64  |234.149.189.184 |simhostb1                  |SUT                 |none    |none       |C0D45692-16C5-5961-B767-1F7D2D88E2BA
|vm_6          |tinyvm              |pico32      |152.158.24.44   |simhosta1                  |SUT                 |none    |none       |492E0B40-ACD5-5988-8E19-5FBC85F40964
|vm_7          |tinyvm              |pico32      |137.66.82.46    |simhosta1                  |SUT                 |none    |none       |0A1BC29D-DE23-5E55-ABD2-E7089817872F
|vm_8          |tinyvm              |pico32      |253.35.67.124   |simhostc1                  |SUT                 |none    |none       |2F373C19-4688-553C-8374-38B9C2CA9A24
|vm_9          |db2                 |platinum64  |53.220.30.29    |simhostc1                  |SUT                 |none    |none       |F1D7ECA0-DA01-5B54-8BB4-6B396E7737EC
|vm_10         |netclient           |iron32      |7.96.222.105    |simhosta2                  |SUT                 |none    |none       |A6939FD1-C4FD-566E-944A-15D401013A19
|vm_11         |fen_hpc             |platinum64  |228.7.18.12     |simhostd1                  |LG                  |none    |none       |1B0160FC-BFFE-5649-AA91-591DADC8ED65
|vm_12         |tinyvm              |iron32      |113.130.128.18  |simhostb1                  |SUT                 |none    |none       |287BAFB5-E242-577D-8F75-3E7F4783DA83
|vm_13         |tinyvm              |iron32      |81.202.79.105   |simhosta1                  |SUT                 |none    |none       |F7804CA2-4B8B-58F5-8DD8-C995DFE6A74F

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.63M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2218246                       
|Total Connections Received                           |8122                          
|Total Commands Processed                             |2517041                       
|Number of Keys                                       |381                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1642496                       
|Data Size                                            |516112                        
|Index Size                                           |433328                        
|Average Object Size                                  |511.508424182                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |12                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |12                            
|VMs ARRIVED                                          |13                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |102                           

 ---------------- VMC-WIDE COUNTERS ----------------
|845D544A-3F3E-5433-BD75-03F09FBB3BE7 (simzone_c) VM RESERVATIONS|3                             
|F1774E78-5022-5568-80E1-9A3EFC329DB1 (simzone_b) VM RESERVATIONS|3                             
|6B2CF2C9-7ECD-535A-9836-6D30EBA8F83F (simzone_d) VM RESERVATIONS|1                             
|A3B38AF9-68E8-5E7C-8050-66F6D9681E02 (simzone_a) VM RESERVATIONS|5                             

