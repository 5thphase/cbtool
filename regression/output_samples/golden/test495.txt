running shell command: "rm -rf /tmp/cb*_was_used_on_execution"....
stdout:
 

The global object "vm_defaults" on Cloud TESTCLOUD was modified:
|"sub-attribute" (key)                |old value                          |new value                          
|execute_script_name                  |execute_on_staging.sh              |/home/msilva/cloudbench/regression/..//regression/scripts/execute_on_staging.sh
Background operation execution success. Operation "vm-attach" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.4,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:172.16.0.4,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,tinyvm,auto,empty,iron32,execute_provision_complete,empty=empty --operation=vm-attach --cn=TESTCLOUD --uuid=5DB01350-2042-5E99-8483-8E8D9124D3DA --daemon". The process id is 4750.

Going to wait until the value on counter "VM ARRIVING" is equal to 0 (currently it is equal to 1) waiting 1.0 seconds between samples, for a maximum of 36000.0 seconds. The counter is assumed to be decreasing.
 status: Counter "VM ARRIVING" equals 1 after 0.0292870998383 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 1.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 2.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 3.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 4.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 5.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 1 after 6.02928709984 seconds (the counter is assumed to be decreasing).
 status: Counter "VM ARRIVING" equals 0 after 7.02928709984 seconds (the counter is assumed to be decreasing).
Waited 7.02928709984 seconds until "VMs ARRIVING" was equal to 0.
running shell command: "ls /tmp/cb*_was_used_on_execution"....
stdout:
 /tmp/cb_vapp_TESTCLOUD_vm_13_5DB01350-2042-5E99-8483-8E8D9124D3DA.json_was_used_on_execution


The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |173.109.254.72  |simhostc14                 |SUT           |private    |none    |none       |A18E4034-B1AC-5D23-9AE9-7CA97E0605FE
|vm_3          |tinyvm              |pico32      |123.141.126.144 |simhostb1                  |SUT           |private    |none    |none       |20354CC2-0A89-5805-B071-F8538A5EC02A
|vm_4          |db2                 |copper32    |253.127.204.119 |simhosta7                  |SUT           |private    |none    |none       |2778D6AB-27BE-585F-92E6-0C0D743472ED
|vm_5          |netclient           |platinum64  |188.169.160.26  |simhostd11                 |SUT           |private    |none    |none       |8948863C-5E9F-5C74-B1D3-2E189666C1FE
|vm_6          |tinyvm              |pico32      |4.175.152.102   |simhosta0                  |SUT           |private    |none    |none       |3F21F98C-965E-5410-A6FF-7BA560059563
|vm_7          |tinyvm              |pico32      |200.32.165.110  |simhosta3                  |SUT           |private    |none    |none       |561A99D7-85B6-5CA8-A194-DA79949322A2
|vm_8          |tinyvm              |pico32      |196.249.101.183 |simhostb9                  |SUT           |private    |none    |none       |B8002E90-5A09-5A32-BFDD-D24CB6C2D371
|vm_9          |db2                 |platinum64  |15.105.189.141  |simhostb6                  |SUT           |private    |none    |none       |549C6E55-907D-556E-BC5F-D083EED1F4F5
|vm_10         |netclient           |iron32      |187.70.101.110  |simhosta2                  |SUT           |private    |none    |none       |894E617D-D473-5DED-9C06-886260CEF173
|vm_11         |fen_hpc             |platinum64  |167.64.154.173  |simhostc7                  |LG            |private    |none    |none       |B279F199-B432-5CAA-8FF2-F0F8094F47AC
|vm_12         |tinyvm              |iron32      |126.164.234.224 |simhostc9                  |SUT           |private    |none    |none       |5799E2B5-9D6D-53A0-8655-4F71A31CB278
|vm_13         |tinyvm              |iron32      |159.76.245.43   |simhosta6                  |SUT           |private    |none    |none       |5DB01350-2042-5E99-8483-8E8D9124D3DA

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |16.65M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12856971                      
|Total Connections Received                           |548673                        
|Total Commands Processed                             |42689562                      
|Number of Keys                                       |806                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |12                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |12                            
|VMs ISSUED                                           |13                            
|VMs ARRIVED                                          |13                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |6                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |222                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9ADC8244-8C1F-5728-8581-9D8825A87B6E (simzone_a) VM RESERVATIONS|2                             
|662A8BAF-6270-52A0-B49B-7E1326CCC8B6 (simzone_c) VM RESERVATIONS|4                             
|8E167E6D-B4E1-531D-A90A-A469EF1067C6 (simzone_b) VM RESERVATIONS|5                             
|B37F024B-5D05-5130-8E9E-0804A1122684 (simzone_d) VM RESERVATIONS|1                             

